File: app/src/main/java/com/example/zdrowie/ui/home/HomeViewModel.java
package com.example.zdrowie.ui.home;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class HomeViewModel extends ViewModel {

    private final MutableLiveData<String> text;

    public HomeViewModel() {
        text = new MutableLiveData<>();
    }

    public LiveData<String> getText() {
        return text;
    }
}

File: app/src/main/java/com/example/zdrowie/java/ui/home/HomeViewModel.kt
package com.example.zdrowie.java.ui.home

import androidx.lifecycle.ViewModel
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.LiveData

class HomeViewModel : ViewModel() {
    private val text: MutableLiveData<String> = MutableLiveData()
    fun getText(): LiveData<String> {
        return text
    }

}